storageClient/storageClient.go

```diff
@@ -267,6 +267,22 @@ func (c *StorageClient) ShareRecordWithGroup(ctx context.Context, params GroupRe
 	return result, err
 }
 
+// GetSharedWithGroup shares a record type for all group members in given group
+func (c *StorageClient) GetSharedWithGroup(ctx context.Context, params ListGroupRecordsRequest) (*ListGroupRecords, error) {
+	var result *ListGroupRecords
+	path := c.Host + storageServiceBasePath + "/groups/share"
+	req, err := e3dbClients.CreateRequest("GET", path, params)
+	if err != nil {
+		return result, err
+	}
+	urlParams := req.URL.Query()
+	urlParams.Set("nextToken", strconv.Itoa(int(params.NextToken)))
+	urlParams.Set("max", strconv.Itoa(int(params.Max)))
```

Comment:
Think about what happens if this isn’t set by the caller, it would default to the 0 value for its type, integer, which would result in no group records being returned? Probably best to only set this value as a query param if it is non zero.